src/components/sidebar/sidebar.component.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import Icon from '../icon/icon';
+import Modal from '../modal/modal';
+import SidebarStyle from './sidebar.style';
+import './sidebar.scss';
+
+class Sidebar extends Modal {
+  /** Returns classes for the component. */
+  get mainClasses() {
+    return classNames(
+      'carbon-sidebar',
```

Comment:
is that class needed?

Reply:
Yes, the class is used also in other component

Reply:
could you leave a comment in a task to rewrite that other component?

Reply:
done